docs/integrations/aws_codebuild.md

```diff
@@ -42,15 +40,43 @@ Running streampipe in CodeBuild uses the same `run-as: codebuild-user` as the in
     run-as: codebuild-user
     commands:
       - ./steampipe --version # for debugging
-      - export STEAMPIPE_WORKSPACE_CHDIR=`pwd`/steampipe-mod-terraform-aws-compliance
+      - export STEAMPIPE_MOD_LOCATION=`pwd`/steampipe-mod-terraform-aws-compliance
       - cd terraform ; ../steampipe check all --output html > steampipe_report.html
 
 ```
 
 Because the `steampipe check all` command returns the number of violations and we want CodeBuild to exit cleanly, we add `on-failure: CONTINUE` to the build phase.
 
-For the actual check, we just need to tell steampipe where to find the mod `export STEAMPIPE_WORKSPACE_CHDIR`, then we change directory to the terraform and run the  `steampipe check all` command. Since steampipe was installed in the parent directory of terraform, we call it as `../steampipe`.
+For the actual check, we just need to tell steampipe where to find the mod `export STEAMPIPE_MOD_LOCATION`, then we change directory to the terraform and run the  `steampipe check all` command. Since steampipe was installed in the parent directory of terraform, we call it as `../steampipe`.
+
+
+## Using Steampipe Cloud
+
+Steampipe can also push snapshots to Steampipe Cloud via AWS CodeBuild. To do this we make a few changes to the buildspec file.
+
+First, we must add the Steampipe Cloud environment variables (stored in Secrets Manager). Add this to the top of the file (before phases):
+```yaml
+env:
+  # Store the Steampipe Cloud host, token and workspace in AWS Secrets Manager
+  secrets-manager:
+    STEAMPIPE_CLOUD_TOKEN: steampipe-cloud:STEAMPIPE_CLOUD_TOKEN
+    WORKSPACE: steampipe-cloud:WORKSPACE
+```
+
+Next, replace the last line of the build with a call to Steampipe Cloud:
+```yaml
+      - cd terraform ; ../steampipe check all --snapshot-location $WORKSPACE --snapshot --snapshot-title "Terraform Report"
+```
+This command will run steampipe and save the output of the check as "Terraform Report" in the specified Workspace. By default, the CLI looks for your steampipe cloud token in the `STEAMPIPE_CLOUD_TOKEN` [environment variable](https://steampipe.io/docs/reference/env-vars/overview).
```

Comment:
I think we should add a little context for how to generate steampipe cloud token.  You probably want to [create a non-expiring token](https://steampipe.io/docs/cloud/profile#tokens), but could alternately run [steampipe login](https://steampipe.io/docs/reference/cli/login) and copy the temporary token (30 day expiration currently)